segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/SystemJournal.java

```diff
@@ -173,10 +241,12 @@ public void initialize() throws Exception {
      * Bootstrap the metadata about storage metadata segments by reading and processing the journal.
      *
      * @param epoch      Epoch of the current container instance.
+     * @param snapshotInfoStore Store that saves {@link SnapshotInfo}.
```

Comment:
this doesn't just "save", does it? You can also use it for retrieval.

Reply:
Fixed.